### DISCUSSION TITLE
Domain verification issue

### DISCUSSION THREAD

**Discussion by filipmyllari** [vercel/community#6648] [Account Issue]
### Question

Hey! We are running into a problem when we are trying to move one of our domains from an old lost account to our new one.

The domain in question is www.tierset.se which we own but when I try to add the verification code to my domain provider in the DNS settings nothing is happening (have left it for a couple of days and still nothing)
"Domain www.tierset.se was added to a different project. Please complete verification to add it to this project instead."

Is there a possibility of moving the domain out of the old account into our new account without being able to sign in to the old one? Evidence can be provided that we own the domain.

Best regards,

Filip

**Comment by versecafe** [comment]
@filipmyllari it should be possible to on your DNS providers side stop any references that point to your old account such as TXT verification records, and then configure for just your new account, if the domain is managed directly with Vercel either purchased through them or transferred to Vercel in full then you'd need to reach out to the support team for help and provide proof of ownership/purchase and they should be able to help you.
